model/model_updater.py

```diff
@@ -58,12 +58,15 @@ async def _get_metadata(self, hotkey: str) -> Optional[ModelMetadata]:
         """Get metadata about a model by hotkey"""
         return await self.metadata_store.retrieve_model_metadata(hotkey)
 
-    async def sync_model(self, hotkey: str, force: bool = False) -> bool:
+    async def sync_model(
+        self, hotkey: str, curr_block: int, retry_stable_metadata: bool = False
```

Comment:
nit: Even with the comment I found this a little more confusing than just "force"

Reply:
Reverted.